test/functional/wiki_controller_test.rb

```diff
@@ -31,6 +31,21 @@ def teardown
     assert assigns(:wikis).each_cons(2).all?{|i,j| "j.node_revisions.title" >= "i.node_revisions.title" }
   end
 
+  test 'should paginate the wikis' do
+    31.times{
+    post :create,
+         params: {
+           uid:   users(:bob).id,
+           title: 'Test',
+           body:  'This is fascinating documentation about balloon mapping.',
+           tags:  'balloon-mapping'
+         }
+    }
```

Comment:
Thanks for creating the test @gaurav2699, I am curious if this request might take a long time to run? 
There is a file where the test database can be preloaded with data, `test/fixtures` and within it, is the `nodes.yml` file  `test/fixtures/nodes.yml` that contains wikis as well as research notes, to create a wiki within the `nodes.yml` file the `type: "page"`.

Reply:
hey @RuthNjeri, thanks for the review. I am aware of nodes.yml but it contained only 10 nodes that are wikis(nodes with type: 'page'). To test if the pagination would work, having just 10 nodes was too less as every page will contain 10 items. Hence I created a few wikis, though I agree 31 is a big number, I will change it to a lesser value.

Reply:
I did not mean that the number is huge, just that a POST request being made 31 times seems like an expensive call as opposed to having prepopulated nodes for this...what do you think?

Reply:
Or should I create few more wikis in the nodes.yml file only, but I think a few other tests depend on the wikis in that hence I didn't feel it was a good idea to touch that.

Reply:
Anyways, I reduced 31 to 6, is that good enough?

Reply:
Anyways, I reduced 31 to 12, is that good enough?

Reply:
Yeah, I agree that this is slower. So should I add few more wikis in the node.yml file itself?

Reply:
Hi @gaurav2699 I think it's okay for now since the number has reduced to 12...